OpenID_Connect/oidc_config.sh

```diff
@@ -61,6 +179,15 @@ if [ $# -gt 0 ] ; then
   print_usage
 fi
 
+if hash jq 2>/dev/null ; then
```

Comment:
should this block be moved to "env.sh"?

Reply:
You are totally right. As the code has an `exit 1`, I wanted to keep them at the same file, but displaying the error message and exiting is OK for this code to be at `env.sh`. :+1: